Database/database_scripts.sql

```diff
@@ -0,0 +1,65 @@
+/* This file can be run to create the necessary objects for
+    a Dao game database */
+
+/* Create the Database */
+CREATE DATABASE DAOdb;
+GO

```

Comment:
GO!  lol I didn't realize this was how you told the database to start

Reply:
"GO" is fun to use. It's not even really necessary most of the time but I like using it when working directly with the structures!